samples/EventImporter/Provider/Domain/Models/Event.cs

```diff
@@ -1,16 +1,22 @@
 using System;
+using Newtonsoft.Json;
 
 namespace Provider.Domain.Models
 {
     public class Event
     {
+        [JsonProperty("eventId")]
         public Guid EventId { get; set; }
+
+        [JsonProperty("timestamp")]
         public DateTime Timestamp { get; set; }
+
+        [JsonProperty("eventType")]
```

Comment:
Ah, here's the problem. Can we remove these please to keep the pacts the same?

Reply:
then the validation fails because it doesn't find the properties, as pact verifier is looking for them in camelCase.

Reply:
Also this is a change on provider side, where as consumer is generating the pact as it was previously with anonymous object where properties are CamelCased, yet the pact ends up camelCased.

Reply:
This is fixed in #351 so can be stripped from this PR